themes.d.ts

```diff
@@ -1,3 +1,6 @@
 import * as light from "./build/ts/themes/light";
 
+type AviaryTheme = typeof light;
```

Comment:
Just confirming that this should work for other themes because it's all the same type?